engines/python/setup/djl_python/test_model.py

```diff
@@ -103,27 +131,85 @@ def create_npz_request(list, key: str = None) -> Input:
     return request
 
 
-def _extract_output(outputs: Output) -> Input:
+def extract_output_as_input(outputs: Output) -> Input:
     inputs = Input()
     inputs.properties = outputs.properties
     inputs.content = outputs.content
     return inputs
 
 
 def extract_output_as_bytes(outputs: Output, key=None):
-    return _extract_output(outputs).get_as_bytes(key)
+    return extract_output_as_input(outputs).get_as_bytes(key)
 
 
 def extract_output_as_numpy(outputs: Output, key=None):
-    return _extract_output(outputs).get_as_numpy(key)
+    return extract_output_as_input(outputs).get_as_numpy(key)
 
 
 def extract_output_as_npz(outputs: Output, key=None):
-    return _extract_output(outputs).get_as_npz(key)
+    return extract_output_as_input(outputs).get_as_npz(key)
 
 
 def extract_output_as_string(outputs: Output, key=None):
-    return _extract_output(outputs).get_as_string(key)
+    return extract_output_as_input(outputs).get_as_string(key)
+
+
+def load_properties(properties_dir):
+    properties = {}
+    properties_file = os.path.join(properties_dir, 'serving.properties')
+    if os.path.exists(properties_file):
+        with open(properties_file, 'r') as f:
+            for line in f:
+                # ignoring line starting with #
+                if line.startswith("#") or not line.strip():
+                    continue
+                key, value = line.strip().split('=', 1)
+                key = key.strip()
+                value = value.strip()
+                properties[key] = value
+    return properties
+
+
+def update_properties_with_env_vars(kwargs):
+    env_vars = os.environ
+    for key, value in env_vars.items():
+        if key.startswith("OPTION_"):
+            key = key[7:].lower()
+            if key == "entrypoint":
+                key = "entryPoint"
+            kwargs.setdefault(key, value)
+    return kwargs
+
+
+class TestHandler:
+
+    def __init__(self,
+                 entry_point: Union[str, ModuleType],
+                 model_dir: str = None):
+        self.serving_properties = load_properties(model_dir)
+        self.serving_properties = update_properties_with_env_vars(
```

Comment:
env var is default value, we should call update_properties_with_env_vars first, then load properties